pkg/cmd/events/tail/tail.go

```diff
@@ -0,0 +1,163 @@
+package tail
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/search"
+	"github.com/algolia/cli/pkg/cmdutil"
+	"github.com/algolia/cli/pkg/config"
+	"github.com/algolia/cli/pkg/iostreams"
+	"github.com/algolia/cli/pkg/printers"
+	"github.com/algolia/cli/pkg/validators"
+
+	_insights "github.com/algolia/algoliasearch-client-go/v3/algolia/insights"
+	region "github.com/algolia/algoliasearch-client-go/v3/algolia/region"
+	"github.com/algolia/cli/api/insights"
+)
+
+const (
+	// DefaultRegion is the default region to use.
+	DefaultRegion = "us"
+
+	// Interval is the interval between each request to fetch events.
+	Interval = 3 * time.Second
```

Comment:
Juste a thought here: do we want to let the user customize this interval with a flag? Not sure it's relevant

Reply:
It's relevant, @shaneafsar had the same suggestion :)
Not sure of the value though, because anyway, if the user try to reduce too much the interval, the API will start to reject the requests.